operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainSpec.java

```diff
@@ -655,8 +655,8 @@ public void setConfigOverrideSecrets(@Nullable List<String> overridesSecretNames
    * @return istioEnabled
    */
   boolean isIstioEnabled() {
-    return Optional.ofNullable(experimental)
-        .map(Experimental::getIstio)
+    return Optional.ofNullable(configuration)
```

Comment:
It's probably easiest for our Verrazzano friends if we keep Istio under Experimental and then add it under Configuration as you've done. If the value under Configuration is present then that is normative -- no merging if Istio is present at both places.